.travis.yml

```diff
@@ -2,21 +2,35 @@ language: android
 
 android:
   components:
+    - tools
     - platform-tools
-    - build-tools-23.0.1
-    - android-23
+    - tools
```

Comment:
Yes.  This tricks travis into accepting the license for the SDK.  See https://github.com/travis-ci/docs-travis-ci-com/issues/779.